src/cryptonote_basic/difficulty.cpp

```diff
@@ -130,10 +130,12 @@ namespace cryptonote {
       cumulative_difficulties.erase(cumulative_difficulties.begin(), cumulative_difficulties.begin() + num_ignored_blocks);
     }
 
+    const uint64_t difficulty_lag = version < 9 ? DIFFICULTY_LAG_V1 : DIFFICULTY_LAG_V9;
```

Comment:
Should that 9 be HF_VERSION_REMOVE_DIFFICULTY_SORT ? Or are those changes two independent things ?

Reply:
They are independent.